scalableminds/webknossos
# Welcome Toast for unregistered Users

### URL of deployed dev instance (used for testing):
- https://___.webknossos.xyz

### Steps to test:
- open the a public tracing (or dataset) while being logged out
- --> welcome toast should appear
- reload --> toast should not appear
- perform `localStorage.clear()` and reload again to see the toast once more

### Issues:
- fixes #5425 

------
